### ISSUE TITLE
Pick a decent icon, then release to Chrome Web Store.

### ISSUE THREAD

**Issue by karlhorky** [jamestalmage/gh-nag#3]
Hi, thanks for the extension! Would you consider releasing this to the Chrome Web Store for easier installation?

**Comment by jamestalmage** [comment]
Yeah, that's the plan. I just whipped it up quick yesterday, I just need to find a decent icon.

**Comment by mrmartineau** [comment]
@jamestalmage have you found an icon yet? I'd love to install this from the Chrome store

**Comment by jamestalmage** [comment]
I am swamped right now, and not much of an icon artist. If anyone wants to contribute an icon, I'm all for it. 

The only thought I had was an icon created by interlocking both the PR and Issue icons. See https://github.com/github/octicons (I am not sure I understand the [license for the icons](http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL), but from my reading I think this would be permitted). I'm not married to the idea, and would certainly entertain any good ideas / contributions.
